### PR TITLE
Get element text using innerText OR textContent

### PR DESCRIPTION
**Pull request by joshski**
Fixes a few tests in firefox

### CODE REVIEW

File: elementTester.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ module.exports = function elementTester(options) {
   }
 
   function getNormalisedText(el) {
-    return el[0].innerText.replace(/ +/g,' ').replace(/ *\n */g,"\n");
+    return (el[0].innerText || el[0].textContent).replace(/ +/g,' ').replace(/ *\n */g,"\n");;
```
Comment (refractalize) [inline elementTester.js]: stray `;`?

**Comment by joshski** [comment]
Actually this is dumb (if innerText is falsy), don't merge yet.

**Comment by dereke** [comment]
so I actually completely rewrite elementTester to make it more extensible (haven't pushed it yet, was just an experiment).
but anyway regarding these firefox failures, once the virtual-dom tests are working I imagine most of the firefox failures will be fixed.. soo if you want to help me over on the #virtual-dom branch feel free :-)

**Comment by dereke** [comment]
oh and I say the firefox tests will probably pass once the virtual-dom branch is working because I am using the jquery api for all dom access... which kinda implies it will work cross browser.

**Comment by dereke** [comment]
complete firefox compatibility has been implemented on the virtual-dom branch that will become part of 2.0
so I am going to close this.

**Pull request closed without merging**